plugins/telemetry/telemetry.h

```diff
@@ -43,6 +43,21 @@ class Telemetry : public PluginBase {
         float relative_altitude_m; /**< @brief Altitude relative to takeoff altitude in metres */
     };
 
+    /**
+     * @brief Position and velocity local type.
+     */
+    struct PositionVelocityLocal {
+        float north_m; /**< @brief Position along north-direction of local frame in meters. */
```

Comment:
I don't think it's clear what is meant with `north_m` and `north_m_s`. 

I think it would make more sense to have the structure like this:
```
struct LocalPositionNED {
    float north_m;
    float east_m;
    float down_m
};
struct VelocityNED {
    float north_m_s;
    float east_m_s;
    float down_m_s;
};
struct LocalPositionAndVelocity {
    LocalPositionNED position;
    VelocityNED velocity;
};
```
and maybe you can even share some of the types with other structs.

Reply:
I added the structure you suggested, but decided against using one of the other structures, despite the available velocity structure (`GroundSpeedNED`). The name GroundSpeedNED is contradicting because `Speed` implies that there is no direction information and the values always have to be positive (in contrast to `velocity`). However, `NED` implies the opposite that the quantity in question is velocity and not speed.